packages/@aws-cdk/aws-lightsail/lib/instance.ts

```diff
@@ -0,0 +1,415 @@
+  /**
+   * redmine
+   */
+  public static readonly REDMINE = Blueprint.of('redmine');
+  /**
+   * nginx
+   */
+  public static readonly NGINX = Blueprint.of('nginx');
+  /**
+   * ghost_bitnami
+   */
+  public static readonly GHOST_BITNAMI = Blueprint.of('ghost_bitnami');
+  /**
+   * django_bitnami
+   */
+  public static readonly DJANGO_BITNAMI = Blueprint.of('django_bitnami');
+  /**
+   * prestashop_bitnami
+   */
+  public static readonly PRESTASHOP_BITNAMI = Blueprint.of('prestashop_bitnami');
+  /**
+   * plesk_ubuntu_18_0_34
+   */
+  public static readonly PLESK_UBUNTU_18_0_34 = Blueprint.of('plesk_ubuntu_18_0_34');
+  /**
+   * cpanel_whm_linux
+   */
+  public static readonly CPANEL_WHM_LINUX = Blueprint.of('cpanel_whm_linux');
+
+  /**
+   * custom blueprint id
+   *
+   * @param id the blueprint id
+   */
+  public static of(id: string) {
+    return new Blueprint(id);
+  }
+  /**
+   * @param id the blueprint id
+   */
+  private constructor(public readonly id: string) {}
+}
+
+/**
+ * The Bundle for the instance.
+ */
+export class Bundle {
+  /**
+   * nano_2_0
+   */
+  public static readonly NANO_2_0 = Blueprint.of('nano_2_0');
+  /**
+   * micro_2_0
+   */
+  public static readonly MICRO_2_0 = Blueprint.of('micro_2_0');
+  /**
+   * small_2_0
+   */
+  public static readonly SMALL_2_0 = Blueprint.of('small_2_0');
+  /**
+   * medium_2_0
+   */
+  public static readonly MEDIUM_2_0 = Blueprint.of('medium_2_0');
+  /**
+   * large_2_0
+   */
+  public static readonly LARGE_2_0 = Blueprint.of('large_2_0');
+  /**
+   * xlarge_2_0
+   */
+  public static readonly XLARGE_2_0 = Blueprint.of('xlarge_2_0');
+  /**
+   * 2xlarge_2_0
+   */
+  public static readonly TWO_XLARGE_2_0 = Blueprint.of('2xlarge_2_0');
+  /**
+   * nano_win_2_0
+   */
+  public static readonly NANO_WIN_2_0 = Blueprint.of('nano_win_2_0');
+  /**
+   * micro_win_2_0
+   */
+  public static readonly MICRO_WIN_2_0 = Blueprint.of('micro_win_2_0');
+  /**
+   * small_win_2_0
+   */
+  public static readonly SMALL_WIN_2_0 = Blueprint.of('small_win_2_0');
+  /**
+   * medium_win_2_0
+   */
+  public static readonly MEDIUM_WIN_2_0 = Blueprint.of('medium_win_2_0');
+  /**
+   * large_win_2_0
+   */
+  public static readonly LARGE_WIN_2_0 = Blueprint.of('large_win_2_0');
+  /**
+   * xlarge_win_2_0
+   */
+  public static readonly XLARGE_WIN_2_0 = Blueprint.of('xlarge_win_2_0');
+  /**
+   * 2xlarge_win_2_0
+   */
+  public static readonly TWO_XLARGE_WIN_2_0 = Blueprint.of('2xlarge_win_2_0');
+
+  /**
+   * custom bundle id
+   *
+   * @param id the instance id
+   */
+  public static of(id: string) {
+    return new Bundle(id);
+  }
+  /**
+   * @param id the bundle id
+   */
+  private constructor(public readonly id: string) {}
+}
+
+/**
+ * The properties of the instance.
+ */
+export interface InstanceProps extends cdk.ResourceProps {
+  /**
+   * The blueprint of the instance.
+   *
+   * @default AMAZON_LINUX_2
+   */
+  readonly blueprint?: Blueprint;
```

Comment:
I think this should be a required property. I think the user should have to think about what type of Instance they want.